common/chunkStatusLogger.go

```diff
@@ -358,17 +344,11 @@ func (csl *chunkStatusLogger) main(chunkLogPath string) {
 	alwaysFlushFromNowOn := false
 	for x := range csl.unsavedEntries {
```

Comment:
comment above this

// We will exit the following for-range loop after CloseLogger() closes the csl.unsavedEntries channel.